src/components/Button/tests/Button.test.tsx

```diff
@@ -220,6 +220,44 @@ describe('<Button />', () => {
     });
   });
 
+  describe('onMouseEnter()', () => {
+    it('is called when the mouse enters button', () => {
+      const onMouseEnterSpy = jest.fn();
+      const button = mountWithAppProvider(
+        <Button onMouseEnter={onMouseEnterSpy} />,
+      );
+      trigger(button.find('button'), 'onMouseEnter');
+      expect(onMouseEnterSpy).toHaveBeenCalledTimes(1);
+    });
+    it('is called when the mouse enters link', () => {
```

Comment:
Could we separate tests with a new line?

Reply:
I was following how the onClick test was set up! Should I still add the space?

Reply:
Ah yeah, spacing is not very consistent in this file. If it's not too much trouble, could you fix this throughout this file?